### PR TITLE
Update complete_cip.html.erb

### PR DESCRIPTION
**Pull request by Lloyd-K**
Timeframe and terminology updated.

https://dfedigital.atlassian.net/browse/CPDRP-912?atlOrigin=eyJpIjoiN2I1NTkwYWVhYjI5NDQxM2E0MmU1ZmMwMTYxNTVmZTgiLCJwIjoiaiJ9

### CODE REVIEW

File: app/views/participants/validations/complete_cip.html.erb [hunk 1]
```diff
@@ -7,10 +7,10 @@
     <h2 class="govuk-heading-m">What happens next</h2>
     <p class="govuk-body">We may need to contact you for more information to complete your registration.</p>
     <% if @participant_profile.ect? %>
-      <p class="govuk-body">Your training materials will be available by the end of August. We’ll email you with a link to access them.</p>
+      <p class="govuk-body">We'll email you a link to access your materials within the next 24 hours.</p>
     <% else %>
       <!-- TODO: CPDRP-886 revert to 48 hours -->
-      <p class="govuk-body">If you need access to training materials, we’ll email you a link soon.</p>
+      <p class="govuk-body">If you need access to materials, we’ll email you a link soon.</p>
```
Comment (ethanmills) [inline app/views/participants/validations/complete_cip.html.erb:13]:
```suggestion
      <p class="govuk-body">If you need access to materials, we’ll email you a link within the next 24 hours.</p>
```

File: app/views/participants/validations/complete_cip.html.erb [hunk 2]
```diff
@@ -7,10 +7,10 @@
     <h2 class="govuk-heading-m">What happens next</h2>
     <p class="govuk-body">We may need to contact you for more information to complete your registration.</p>
     <% if @participant_profile.ect? %>
-      <p class="govuk-body">Your training materials will be available by the end of August. We’ll email you with a link to access them.</p>
+      <p class="govuk-body">We'll email you a link to access your materials within the next 24 hours.</p>
     <% else %>
       <!-- TODO: CPDRP-886 revert to 48 hours -->
```
Comment (ethanmills) [inline app/views/participants/validations/complete_cip.html.erb:12]:
```suggestion
```

**Review by ethanmills — APPROVED** [review]

**Comment by tonyheadford** [comment]
Specs are failing due to the content change. I'll update them for you @Lloyd-K

File: app/views/participants/validations/complete_cip.html.erb [hunk 3]
```diff
@@ -7,10 +7,9 @@
     <h2 class="govuk-heading-m">What happens next</h2>
     <p class="govuk-body">We may need to contact you for more information to complete your registration.</p>
     <% if @participant_profile.ect? %>
-      <p class="govuk-body">Your training materials will be available by the end of August. We’ll email you with a link to access them.</p>
+      <p class="govuk-body">We'll email you a link to access your materials within the next 24 hours.</p>
```
Comment (tonyheadford) [inline app/views/participants/validations/complete_cip.html.erb:10]:
```suggestion
      <p class="govuk-body">We’ll email you a link to access your materials within the next 24 hours.</p>
```

**Pull request merged**